lstchain/reco/utils.py

```diff
@@ -201,17 +201,18 @@ def get_event_pos_in_camera(event, tel):
     -------
     (x, y) (float, float): position in the camera
     """
-    array_pointing = HorizonFrame(alt=event.mcheader.run_array_direction[1],
+    array_pointing = AltAz(alt=event.mcheader.run_array_direction[1],
                                   az=event.mcheader.run_array_direction[0])
-    event_direction = HorizonFrame(alt=event.mc.alt.to(u.rad),
+    event_direction = AltAz(alt=event.mc.alt.to(u.rad),
```

Comment:
this should become:
```python
event_direction = SkyCoord(alt=event.mc.alt, az=event.mc.az, frame=horizon_frame)
```
And I don't think you need to do any change to u.rad

Reply:
so you define the `horizon_frame` just once and then you re-use it where you need.